workflow-runtime/src/commonMain/kotlin/com/squareup/workflow1/RenderWorkflow.kt

```diff
@@ -204,6 +207,8 @@ public fun <PropsT, OutputT, RenderingT> renderWorkflowIn(
       renderingsAndSnapshots.value = nextRenderAndSnapshot
       // And emit the Output.
       sendOutput(actionResult, onOutput)
+      // Yield if there are any side effects (they were launched lazily) that need starting.
+      yield()
```

Comment:
Are you sure we shouldn't `yield()` first, before we allow the UI to update?

Reply:
in order to get the side effects started before then you mean? Or because we don't want the collectors of the StateFlow of renderings to be in contention for dispatch at the same time?

Reply:
The former was what I was thinking of -- seems like a side effect should happen, or start happening, before / while the UI is updating.

Reply:
Thought exercise: what happens if something on the receiving end of `sendOutput` kills off the workflow runtime before that `yield()` happens (if it that's possible). Would we be surprised that the last declared side effects were never run? I think we would.

Reply:
all legitimate points. I adjusted the ordering in the updated PR.